storyblok/PTRFooter.vue

```diff
@@ -12,7 +12,7 @@ const props = defineProps<PropTypes>()
 
 <template>
   <footer v-editable="props.blok" class="bg-primaryDark pt-8 sm:pt-16">
-    <div class="container px-2 sm:px-8">
+    <div class="container px-2 sm:px-8 pb-0">
```

Comment:
`pb-0` here is unneeded
```suggestion
    <div class="container px-2 sm:px-8">
```